docs/guide.rst

```diff
@@ -87,6 +87,21 @@ Here's how you can deploy new contracts:
     :dedent: 4
 
 
+Deploying new contracts with Universal Deployer Contract (UDC)
+--------------------------------------------------------------
+
+The best way of deploying contracts on Starknet is to use UDC. starknet.py assumes you use an implementation compatible
```

Comment:
This statement is not true

Reply:
Maybe it's most "universal" as it doesn't require any special methods in the contract unlike deploy_account.

Reply:
Still, i think we should not make those kind of statements in the docs - syscall is the only supported way and this is a wrapper around that

Reply:
Syscall and deploy_account transaction - it can deploy any contract not only accounts. It's not very practical though.

Reply:
We can say that using UDC is a way of deploying contracts if you already have an account.